src/components/FormElements/PubKeyPicker.tsx

```diff
@@ -21,6 +21,7 @@ export const PubKeyPicker = ({
   placeholder = "Ex: GCEXAMPLE5HWNK4AYSTEQ4UWDKHTCKADVS2AHF3UI2ZMO3DPUSM6Q4UG",
   value,
   error,
+  readOnly,
```

Comment:
Ideally I'd like to have a copy by default on click functionality. I didn't want to add too much to the original component though. Should I just create its own component? @quietbits